pulsar-client/src/main/java/org/apache/pulsar/client/impl/PulsarClientImpl.java

```diff
@@ -860,6 +860,10 @@ public boolean isClosed() {
 
     @Override
     public synchronized void updateServiceUrl(String serviceUrl) throws PulsarClientException {
+        if (serviceUrl != null && serviceUrl.equals(conf.getServiceUrl())) {
+            log.info("Ignore service URL to {}，because it is the same as the current service URL!", serviceUrl);
```

Comment:
`Ignore service URL` -> 'Ignore update service URL' ?

Reply:
![image](https://user-images.githubusercontent.com/19296967/155494985-e03cd038-e4e0-4124-a3b9-c39f5d06be4c.png)
Aha, I made a mistake, here is the judgment of whether the url is equal.  I will close this PR